benchmarks/benchmarks.py

```diff
@@ -62,6 +63,16 @@ def time_convex_hull(self):
     def time_delaunay_triangles(self):
         pygeos.delaunay_triangles(self.points)
 
+    def time_box(self):
+        pygeos.box(*np.hstack([self.coords, self.coords + 100]).T)
+
+    def time_box_prev(self):
```

Comment:
If we keep it, I would call it `time_box_from_rings` or something like that

Reply:
We can delete it now.